torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -3502,6 +3502,19 @@ at::Tensor& AtenXlaType::zero_(at::Tensor& self) {
   return self;
```

Comment:
Can we add a light weight python test to make sure when `item` call is issued, the counter is correctly incremented corresponding to`XLA_SYNC_BEFORE_ITEM_CALL`.